### ISSUE TITLE
Update Authors & Acknowledgements and Bibliography for v0.5

### ISSUE THREAD

**Issue by gefux** [tempoCollaboration/OQuPy#104]
There will be lot's of new contributors for version 0.5.
We should then update the list of contributers and papers.

**Comment by piperfw** [comment]
Oh good, saves me having to make a new issue. Note several of the preprints have since become publications. Do we want to change those for 0.5? 
I would also be tempted to update the dates to reflect the latest publication dates e.g. 
```
- **[Strathearn2019]** Strathearn,
  *Modelling Non-Markovian Quantum Systems Using Tensor Networks*,
  `Springer Theses <http://link.springer.com/10.1007/978-3-030-54975-6>`_
  (2020).
```
should really be `Strathearn2020`

**Comment by gefux** [comment]
Fixed with PR #135
